formapp/app.py

```diff
@@ -48,6 +48,11 @@
 import time, math
 import threading
 import logging
+
+#This import allows for the time to be printed to console. If you need to add a new print line the 2nd line is an example of how to use the logFile() function to make it print to console and to the log file
+import datetime
+#logFile("Info", "Bot successful login. Made them a guest: " + username)
```

Comment:
Remove any code that isn't in use. Comment why code is commented out if it's required.

Reply:
The code is commented how because it is an example of how to use the function as mentioned in the comment above

Reply:
Removed my original reply cause I thought I made an error in my comment but, the code is commented out because, 1 The function wasn't declared yet, 2 it's mentioned as an example in the comment above it

Reply:
Ok. I misread it. Could you move it to the the first line of the function declaration. We don't need the datetime comment, because it's a default python library that most programmers will quickly become familiar with.

Thanks.

Reply:
Can do. Once I get home that is.